.changeset/twenty-maps-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+feat: Add urls to the first and the last page in the Page interface
```

Comment:
```suggestion
Adds two new values to the [pagination `page` prop](https://docs.astro.build/en/reference/api-reference/#the-pagination-page-prop): `page.first` and `page.last` for accessing the URLs of the first and last pages.
```
Maybe something like this? When new options are added, I do always think it's helpful if the changeset message mentions them by name!